code/backend/src/main/java/server/model/EngagementContract.java

```diff
@@ -67,14 +70,57 @@ public EngagementContract(String id,
         this.currencyCode = currencyCode;
         this.timeMaterialTerms = timeMaterialTerms;
         this.poRefNum = poRefNum;
-        this.hourlyRate = hourlyRate;
+        this.dollarRate = dollarRate;
         this.hrPositionRole = hrPositionRole;
         this.hrPayGrade = hrPayGrade;
         this.mainSkill = mainSkill;
         this.rehire = rehire;
         this.hiringManager = hiringManager;
     }
 
+    public int getMonthlyCost(Calendar thisDate) {
+        final int HOURS_IN_DAY = 24;
```

Comment:
We should estimate for 40 hr work week tbh. That might be easier to implement as well.
Then for hourly we could multiply hourly cost by 160 to get monthly cost.
OR multiply daily cost by 20